### ISSUE TITLE
Cannot load SVG images with `GM_getResourceURL` and `GM_addStyle`

### ISSUE THREAD

**Issue by simkoG** [Tampermonkey/tampermonkey#1592]
I cannot set an SVG image as background to an element with `GM_getResourceURL` and `GM_addStyle`. The image won't load for SVG files but works perfectly with PNGs. 

## Expected Behavior
Load SVG images with `GM_getResourceURL` and `GM_addStyle`. 

## Actual Behavior
If you `@grant` `GM_getResourceURL ` and `GM_addStyle` and load and SVG file with `GM_getResourceURL` from a `@resource` url, SVG images will not render properly. They will be empty. 

**Page without the userscript**: 
<img src="https://user-images.githubusercontent.com/10430702/189620476-8899f107-4db8-402f-87f5-71078b555dee.png" width="300px" height="auto">

**Page with userscript**: 
<img src="https://user-images.githubusercontent.com/10430702/189620598-3185f10c-7737-43a2-a3ee-e7c9da9110f8.png" width="300px" height="auto">

You can see, that the second image (which is a png) is replaced with a newer one, but the first is missing after the script loaded. 

## Specifications

- Google Chrome, Version 105.0.5195.102 (Official Build) (64-bit)
- Tampermonkey v4.16.1
- OS: Windows 10, Version 21H2 (OS Build 19044.1949)

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         SVG test
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        *://*/a.html
// @resource     EMOJI_SVG https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/1f36d.svg
// @resource     EMOJI_PNG https://raw.githubusercontent.com/twitter/twemoji/master/assets/72x72/1f36d.png
// @grant        GM_getResourceURL
// @grant        GM_addStyle
// ==/UserScript==

(function() {
    'use strict';

    const svg = GM_getResourceURL( "EMOJI_SVG" );
    GM_addStyle(`

    .icon.svg { background-image: url( "${svg}" ); }

    `);

    const png = GM_getResourceURL( "EMOJI_PNG" );
    GM_addStyle(`

    .icon.png { background-image: url( "${png}" ); }

    `);
})();
```

## Sample page to demonstrate

```html
<!doctype>
<html>
<head>
  <meta charset="utf-8">
  <title>SVG</title>
  
  <style>
  .icon {
    width: 4.5rem;
    height: 4.5rem;
    display: inline-block;
    margin: 2rem;
    background-size: 100%;
    background-repeat: no-repeat;
    background-position: center;
  }
  
  .icon.svg {
    background-image: url(https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/1f41e.svg);
  }
  
  .icon.png {
    background-image: url(https://raw.githubusercontent.com/twitter/twemoji/master/assets/72x72/1f41e.png);
  }
  </style>
</head>

<body>
  <span class="icon svg"></span>
  <span class="icon png"></span>
</body>
</html>
```

**Comment by derjanb** [comment]
Should be fixed at 4.18.6168 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

**Comment by simkoG** [comment]
@derjanb Confirmed. Bug is fixed in 4.18.6168 version.
